federationapi/routing/invite.go

```diff
@@ -78,7 +77,7 @@ func InviteV1(
 ) util.JSONResponse {
 	roomVer := gomatrixserverlib.RoomVersionV1
 	body := request.Content()
-	event, err := roomVer.NewEventFromTrustedJSON(body, false)
+	event, err := gomatrixserverlib.MustGetRoomVersion(roomVer).NewEventFromTrustedJSON(body, false)
```

Comment:
Should we add a comment here explaining why this MustGetRoomVersion is acceptable here? In case we move the hardcoded room version above later on

Reply:
Sure!